modules/core/src/v2/coins/abstractUtxoCoin.ts

```diff
@@ -233,6 +245,34 @@ export abstract class AbstractUtxoCoin extends BaseCoin {
     return this._network;
   }
 
+  static blockExplorer: UTXOBlockExplorer = {
+    baseUrl: "https://api.blockchair.com",
+
+    async getAddressUnspent (addressBase58: string): Bluebird<UnspentInfo[]> {
+      // make a call to blockchair apis
+      const url = `${AbstractUtxoCoin.blockExplorer.baseUrl}/${this.getFullName()}/outputs?addresses=${addressBase58}`;
+      try {
+        const res = await request.get(url);
+        if (res.hasOwnProperty('data')) {
+          const unspentInfos: UnspentInfo[] = res.data.map(function processUnspent(unspent) {
+            return {
+              address: addressBase58,
+              amount: unspent.output_total,
+              txid: unspent.hash,
+              n: unspent.index,
+            }
+          });
+          return unspentInfos;
+        }
+      } catch (e) {
+
+      }
```

Comment:
This empty catch block will hide any errors you ever produce here. Either you should specify which error you want to ignore as benign and throw any others, or remove the try-catch altogether, so that errors are visible.